### ISSUE TITLE
How to distinguish dhcp route delete for default gateway

### ISSUE THREAD

**Issue by nalincode** [vishvananda/netlink#883]
Hi

I subscribed to netlink channel  and got the below details from kernel when add the dhcp route.

if err := vnetlink.RouteSubscribeWithOptions(ch, done, vnetlink.RouteSubscribeOptions{
               ListExisting: true},
       );

logs :  Gateway [10.37.32.1] Priority[1003] Protocol[16] update type[25] 

[admin@kvm-x86-64]# route -n
Kernel IP routing table
Destination     Gateway         Genmask         Flags Metric Ref    Use Iface
0.0.0.0         10.37.32.1      0.0.0.0         UG    1003   0        0 ma1

But while deleting the dhcp route which is dest ip 0.0.0.0 and gateway ip 10.37.32.1  , how to identify the default route is deleted?
Because I got below details from the channel . I  received gatewayip as nil  from netlink channel which is incorrect right ? 
Basically i want to find the delete request of dhcp is for the above add dhcp route .  And for add request the priority and protocol varies from delete request of the route.

logs: Gateway [<nil>] Priority[0] Protocol[2] update type[25]
